crates/core/tedge_api/src/health.rs

```diff
@@ -105,54 +90,6 @@ pub fn health_check_topics(daemon_name: &str) -> TopicFilter {
     .expect("Invalid topic filter")
 }
 
-pub async fn send_health_status(responses: &mut impl PubChannel, daemon_name: &str) {
-    let health_message = health_status_up_message(daemon_name);
-    let _ = responses.send(health_message).await;
-}
-
-pub fn health_status_down_message(daemon_name: &str) -> Message {
-    Message {
-        topic: Topic::new_unchecked(&format!("tedge/health/{daemon_name}")),
-        payload: json!({
-            "status": "down",
-            "pid": process::id()})
-        .to_string()
-        .into(),
-        qos: mqtt_channel::QoS::AtLeastOnce,
-        retain: true,
-    }
-}
-
-pub fn health_status_up_message(daemon_name: &str) -> Message {
-    let clock = Box::new(WallClock);
-    let timestamp = clock
-        .now()
-        .format(&time::format_description::well_known::Rfc3339);
-    match timestamp {
-        Ok(time_stamp) => {
-            let health_status = json!({
-                "status": "up",
-                "pid": process::id(),
-                "time": time_stamp,
-            })
-            .to_string();
-            let response_topic_health =
-                Topic::new_unchecked(format!("tedge/health/{daemon_name}").as_str());
-
-            Message::new(&response_topic_health, health_status)
-                .with_qos(mqtt_channel::QoS::AtLeastOnce)
-                .with_retain()
-        }
-        Err(e) => {
-            let error_topic = Topic::new_unchecked("tedge/errors");
-            let error_msg = format!(
-                "Health message: Failed to convert timestamp to Rfc3339 format due to: {e}"
-            );
-            Message::new(&error_topic, error_msg).with_qos(mqtt_channel::QoS::AtLeastOnce)
-        }
-    }
-}
-
 pub fn is_bridge_health(topic: &str) -> bool {
     if topic.starts_with("tedge/health") {
```

Comment:
This has to be updated to handle `te` health topics.

Reply:
This is used only by AWS and Azure mappers, which will be done in another PR.